modules/ossm-installation-activities.adoc

```diff
@@ -5,14 +5,13 @@
 [id="ossm-installation-activities_{context}"]
 = {ProductName} installation activities
 
-The {ProductName} installation process creates two different projects (namespaces):
+The {ProductName} installation process creates the `openshift-operators` project (also known as a namespace) with pods for:
 
-* istio-operator project (1 pod)
-* istio-system project (17 pods)
-* kiali-operator project (1 pod)
-* observability project (1 pod)
+* {ProductShortName}
+* Kiali
+* Observability (Jaeger)
 
-You first create a Kubernetes _Operator_. This Operator defines and monitors a _custom resource_ that manages the deployment, updating, and deletion of the {ProductShortName} components.
+You first deploy the {ProductName} Operator. This Operator defines and monitors a _custom resource_ that manages the deployment, updating, and deletion of the {ProductShortName} components.
 
 Depending on how you define the custom resource file, you can install one or more of the following components when you install the {ProductShortName}:
```

Comment:
We say "Depending on how you define the custom resource file, you can install one or more of the following components when you install the Service Mesh"

Does this imply that Istio is optional?

Reply:
I rearranged this section and it should be more clear now.